services/frontend/src/scenes/ProjectDetails/ProjectDetailsLayout.js

```diff
@@ -184,4 +161,17 @@ const Changelog = ({sectionName}) => (
     </>
 );
 
+const ContributorBadge = ({name, role}) => {
+    return (
+        <div className="project-contributors-badge">
+            <Tooltip title={role} placement={"top"}>
+                <Avatar className="project-contributors-badge-picture">
+                    {name[0]}
```

Comment:
The Avatar should show the initials of the profile's first and last names - not just the initial of their first name. The logic for generating the initials can be found in `scenes/Profile/Profile.js`. It should be refactored together into the new `UserAvatar` component mentioned above, such that the only thing the component takes in is the user's full name.